packages/itwinui-css/src/slider/slider.scss

```diff
@@ -102,30 +109,34 @@ $tick-height: $iui-baseline; // 12px
     cursor: not-allowed;
     background-color: var(--iui-color-background-4);
     border-color: var(--iui-color-background-4);
+    @media (forced-colors: active) {
+      background-color: GrayText;
+      border-color: GrayText;
+    }
   }
 
   .iui-slider-track {
-    background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+    background-color: var(--iui-color-foreground-4);
   }
 
   .iui-slider-min,
   .iui-slider-max,
   .iui-slider-tick {
     user-select: none;
-    color: var(--iui-text-color-muted);
+    color: var(--iui-color-foreground-4);
     @media (forced-colors: active) {
       color: GrayText;
     }
   }
 }
 
 @mixin iui-slider-horizontal {
-  min-height: $iui-baseline * 2;
+  min-height: calc(var(--iui-size-s) * 2);
 
   .iui-slider-rail {
     width: 100%;
     height: $track-height;
-    top: $iui-sm;
+    top: var(--iui-size-s);
   }
 
   .iui-slider-thumb {
```

Comment:
For the thumb positioning problem mentioned [here](https://github.com/iTwin/iTwinUI/pull/718#issuecomment-1213274552), I think it is because `$track-height + 1` is producing an unexpected result in the actual webpage's CSS.

#### Selected lines (slider.scss):

![image](https://user-images.githubusercontent.com/45748283/184423415-63ccd849-e33e-4a54-882c-f76491fdc662.png)

#### Declaration of $track-height (slider.scss):

![image](https://user-images.githubusercontent.com/45748283/184423711-7854f377-0d40-4991-98a1-3b82fbc49fe5.png)

#### What appears in the browser:

![image](https://user-images.githubusercontent.com/45748283/184423872-1ee4f275-9787-4114-bfa0-2b81033bc0a0.png)

Here there is an expected 1 that is causing a problem. It works fine when we replace it in the browser with the calculated value of 5px.

Reply:
Good catch, should be fixed now